scripts/pi-hole/php/header.php

```diff
@@ -537,8 +537,8 @@ function pidofFTL()
                   </a>
                   <ul class="treeview-menu">
                     <li>
-                        <a href="#" id="pihole-disable-permanently">
-                            <i class="fa fa-stop"></i> <span>Permanently</span>
+                        <a href="#" id="pihole-disable-indefinitely">
```

Comment:
You are not supposed to change the ID.

Reply:
At least, not without changing this line:

https://github.com/pi-hole/AdminLTE/blob/devel/scripts/pi-hole/js/footer.js#L158